bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/binding/BaseThingHandler.java

```diff
@@ -578,9 +579,12 @@ protected boolean isLinked(String channelId) {
      * Returns whether the thing has already been initialized.
      *
      * @return true if thing is initialized, false otherwise
+     * @deprecated use {@link ThingHandlerHelper#isHandlerInitialized(Thing)} or
+     *             {@link ThingHandlerHelper#isHandlerInitialized(ThingHandler)}
      */
+    @Deprecated
     protected boolean thingIsInitialized() {
```

Comment:
I marked it deprecated, but do we need to keep it at all?
At least a handler is initialized, not the thing (regardless that the information correlates) -- correct?